dataset/dataset.go

```diff
@@ -350,12 +361,36 @@ func (c *Client) GetInstance(ctx context.Context, userAuthToken, serviceAuthToke
 		return
 	}
 
-	b, err := ioutil.ReadAll(resp.Body)
+	b, err = ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return
+	}
+
+	return
+}
+
+// GetInstanceDimensionsByBytes returns a list of dimensions for an instance as bytes from the dataset api
+func (c *Client) GetInstanceDimensionsByBytes(ctx context.Context, userAuthToken, serviceAuthToken, instanceID string) (b []byte, err error) {
```

Comment:
In filterAPI client, I did something similar, but with the naming `...Bytes` instead of `...ByBytes`. Maybe it would be a good idea to have the same convention? I can modify FilterAPI if we want to use `ByBytes` or we can rename this function to `GetInstanceDimensionsBytes`

Reply:
I'll change it to meet your convention